### DISCUSSION TITLE
Comando PLAY para reprodução de músicas não funciona.

### DISCUSSION THREAD

**Discussion by wilbor** [robhagemans/pcbasic#179] [Q&A]
Hello! I currently use the Ubuntu 20.04 system and I'm having trouble running the 'PLAY'MB' command and the 'PLAY expression' command. When executing such a command the interpreter immediately shows a blue-screen error message and closes the interpreter. I want to play music in games in the background. Does anyone have any idea why this error occurs? I really appreciate it.

**Comment by Marrin** [comment]
@wilbor Can you please show a minimal example that has this behaviour.  And if the windows closes, please start pcbasic from a console and also copy the messages/traceback here.

Which version of PC-BASIC are you running?  Please make sure to test with the newest version.

Also this might be better suited as an Issue instead.
